modules/samples/product/src/main/resources/cleanup-scripts/postgresql_clean.sql

```diff
@@ -0,0 +1,57 @@
+CREATE OR REPLACE FUNCTION cleanInstances(bpel_inst_state INT, lastActive timestamp)
```

Comment:
It is possible. I added it in https://github.com/wso2/product-bps/pull/87. Thank You.